__tests__/unit/repositories/masking.repository.test.ts

```
@@ -0,0 +1,213 @@
+
+      const result = await findMasksWithFiltersInDB('WHERE status = ANY($1) AND txtp ILIKE $2', 3, [
+        ['STATUS_01_IN_PROGRESS'],
+        '%pain%',
+        10,
+        0,
+      ]);
+
+      expect(result).toEqual({ result: [] });
+      expect(mockHandlePostExecuteSqlStatement).toHaveBeenCalledWith(
+        expect.objectContaining({
+          text: expect.stringContaining('WHERE status = ANY($1) AND txtp ILIKE $2'),
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)